### PR TITLE
CORTX-31968 Add support for K8s Release v0.7.0

### PR DESCRIPTION
**Pull request by shailesh-vaidya**
# Problem Statement
- Add support for K8s Release v0.7.0

# Design
-  For Bug, Describe the fix here.
-  For Feature, Post the link for design

# Coding
   Checklist for Author
-  [x] Coding conventions are followed and code is consistent

# Testing 
  Checklist for Author
- [ ] Unit and System Tests are added
- [ ] Test Cases cover Happy Path, Non-Happy Path and Scalability
- [ ] Testing was performed with RPM
- [x] Jenkins pipelines used for testing - https://eos-jenkins.colo.seagate.com/job/Cortx-Kubernetes/job/deploy-cortx-k8-cluster/1478/consoleFull

# Impact Analysis
  Checklist for Author/Reviewer/GateKeeper
- [ ] Interface change (if any) are documented
- [ ] Side effects on other features (deployment/upgrade)
- [ ] Dependencies on other component(s)

# Review Checklist 
  Checklist for Author
- [x] JIRA number/GitHub Issue added to PR
- [x] PR is self reviewed
- [x] Jira and state/status is updated and JIRA is updated with PR link
- [x] Check if the description is clear and explained

# Documentation
  Checklist for Author
- [ ] Changes done to WIKI / Confluence page / Quick Start Guide

### CODE REVIEW

**Comment by nitisdev** [comment]
Single node: https://eos-jenkins.colo.seagate.com/job/Cortx-Automation/job/RGW/job/setup-cortx-rgw-cluster/6185/console

**Review by nitisdev — APPROVED** [review]

**Review by pankajgborole — DISMISSED** [review]
Approved

**Review by nitisdev — APPROVED** [review]

File: solutions/kubernetes/cortx-deploy.sh [hunk 1]
```diff
@@ -49,7 +49,7 @@ fi
 
 function check_params() {
     if [ -z "$CORTX_SCRIPTS_REPO" ]; then echo "CORTX_SCRIPTS_REPO not provided. Using default: Seagate/cortx-k8s ";CORTX_SCRIPTS_REPO="Seagate/cortx-k8s"; fi
-    if [ -z "$CORTX_SCRIPTS_BRANCH" ]; then echo "CORTX_SCRIPTS_BRANCH not provided. Using default: v0.6.0";CORTX_SCRIPTS_BRANCH="v0.6.0"; fi
+    if [ -z "$CORTX_SCRIPTS_BRANCH" ]; then echo "CORTX_SCRIPTS_BRANCH not provided. Using default: v0.6.0";CORTX_SCRIPTS_BRANCH="v0.7.0"; fi
```
Comment (nitisdev) [inline solutions/kubernetes/cortx-deploy.sh:52]: Please change v0.6.0 > v0.7.0 in echo statement.

**Review by nitisdev — APPROVED** [review]

**Comment by shailesh-vaidya** [comment]
> please update the version to 0.7.0

Done

File: solutions/kubernetes/cortx-deploy.sh (hunk 1 above)
Comment (shailesh-vaidya) [reply solutions/kubernetes/cortx-deploy.sh]: Done

File: solutions/kubernetes/cortx-deploy-functions.sh [hunk 2]
```diff
@@ -151,7 +150,6 @@ function update_solution_config(){
         yq e -i '.solution.common.resource_allocation.zookeeper.resources.limits.cpu = "1000m"' solution.yaml
 
         yq e -i '.solution.common.resource_allocation.kafka.storage_request_size = "8Gi"' solution.yaml
-        yq e -i '.solution.common.resource_allocation.kafka.log_persistence_request_size = "8Gi"' solution.yaml
```
Comment (gauravchaudhari02) [inline solutions/kubernetes/cortx-deploy-functions.sh:154]: @shailesh-vaidya, not able to see this removal in latest services framework? from where did we get this?

**Review by gauravchaudhari02 — APPROVED** [review]

File: solutions/kubernetes/cortx-deploy-functions.sh (hunk 2 above)
Comment (shailesh-vaidya) [reply solutions/kubernetes/cortx-deploy-functions.sh:154]: This is added as part of https://github.com/Seagate/cortx-k8s/pull/233/

**Pull request merged**